docs/start/smesher/trusted_state_quicksync.md

```diff
@@ -0,0 +1,45 @@
+---
+id: trusted_state_quicksync
+title: Trusted State aka Quicksync
+---
+
+## Quicksync
+
+In response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, Quicksync.
+
+Before reading any further, please read the announcement about Quicksync [here](https://spacemesh.io/blog/making-sync-faster/).
+
+### Getting the archive automatically
+
+#### Smapp
+
+Smapp will propose to run Quicksync automatically.
+
+#### CLI
+
+We prepared a simple tool called [quicksync-rs](https://github.com/spacemeshos/quicksync-rs) that simplifies the process of getting the archive.
+
+### Getting the archive manually
+
+The most recent archive for a given version can be found at: `https://quicksync.spacemesh.io/<version>/latest.zst`. This URL will always redirect to the most recent archive for the given version.
+
+The archive is compressed with `zstd` and can be decompressed using the following command: `zstd -d --long=31 o state.sql latest.sql.zst`. Note that the decompressed archive should be placed in the node data directory of the node.
+
+Doing all of this using the CLI would look like the following (assuming bash shell):
+```
+Select your spacemesh version
+SM_VERSION=v1.6.0
+SM_DATA_DIR=/opt/go-spacemesh/
+
+# Create a tmp dir
+TMP_DIR=$(mktemp -d)
+
+# Download quicksync
```

Comment:
Do we mean "Download `state.sql`? I don't see the quicksync CLI tool being used here at all.

Reply:
yes it's state.